example/spec/parallel/cnmWorkflow/LambdaEventSourceSpec.js

```diff
@@ -70,7 +70,7 @@ describe('When adding multiple rules that share a kinesis event stream', () => {
     // clean up stack state added by test
     console.log(`\nCleaning up stack & deleting test stream '${streamName}'`);
     await deleteRules(testConfig.stackName, testConfig.bucket, rulesToDelete, ruleSuffix);
-    await deleteExecution({ prefix: testConfig.stackName, executionArn: executionArn });
+    await waitForExecutionAndDelete(testConfig.stackName, executionArn, 'completed');
```

Comment:
Are these the same test fixes that you made in main? Either way, thanks for handling them.

Reply:
Yeah, seems like you needed to make all of these changes with the knex 2.x update. Same as https://github.com/nasa/cumulus/pull/3222